Makefile

```diff
@@ -1,3 +1,8 @@
+# Clean all generated SDK files
```

Comment:
Why only the `SDK` files?  Should it not clean the other build artifacts as well?